www/docs/server/data-transformers.md

```diff
@@ -51,11 +50,73 @@ export default trpcNext.createNextApiHandler({
 });
 ```
 
+## Different transformers for upload and download
+
+If a transformer should only be used for one directon or different transformers should be used for upload and download (e.g. for performance reasons), you can provide individual transformers for upload and download. Make sure you use the same combined transformer everywhere.
+
+### Howto
+
+For demonstration purpose [superjson](https://github.com/blitz-js/superjson) should be used for uploading and faster but insecure [devalue](https://github.com/Rich-Harris/devalue) for downloading data.
```

Comment:
```suggestion
Here [superjson](https://github.com/blitz-js/superjson) is be used for uploading and [devalue](https://github.com/Rich-Harris/devalue) for downloading data, because devalue is a lot faster but insecure to use on the server. 
```